tools/api.py

```diff
@@ -0,0 +1,414 @@
+"""Produce a list of API endpoints that are dialed by AoU endpoints.
```

Comment:
I would rather go for a more descriptive file name than `api`? Maybe `generate-api-endpoint-mappings.py` or something?

Reply:
Good idea, will rename.